tests/test_examples.py

```diff
@@ -33,7 +33,11 @@ def test_mnist():
 
 def test_pylearn2():
     filename = 'unittest_markov_chain'
-    pylearn2_test('train', filename, 0, 3, False)
+    try:
+        pylearn2_test('train', filename, 0, 3, False)
+    except OSError:
```

Comment:
It ended up in an infinite recursion at one point, and for some reason it kept crashing my tests. I later found out that it is because of [this terrible line](https://github.com/lisa-lab/pylearn2/blob/master/pylearn2/utils/serial.py#L149). I silence the Pylearn2 logs in the tests, so I never saw that it was waiting for 5 minutes... I'll have another look now that it's done.